src/commcare_cloud/fab/utils.py

```diff
@@ -173,34 +172,32 @@ def _get_github_credentials(message, force=False):
             "    $ cp {project_root}/config.example.py {project_root}/config.py\n"
             "Then edit {project_root}/config.py"
         ).format(project_root=PROJECT_ROOT))
-        username = input('Github username or token (leave blank to skip): ') or None
-        password = getpass('Github password: ') if username else None
-        GITHUB_CREDENTIALS = (username, password)
+        GITHUB_TOKEN = getpass('Github Token: ')
     else:
-        GITHUB_CREDENTIALS = (GITHUB_APIKEY, None)
-    return GITHUB_CREDENTIALS
+        GITHUB_TOKEN = GITHUB_APIKEY
+    return GITHUB_TOKEN
 
 
-def get_github_credentials(message=None, force=False):
+def get_github_token(message=None, force=False):
     if not message:
         message = "This deploy script uses the Github API to display a summary of changes to be deployed."
         if env.tag_deploy_commits:
             message += (
                 "\nYou're deploying an environment which uses release tags. "
                 "Provide Github auth details to enable release tags."
             )
-    return _get_github_credentials(message, force)
+    return _get_github_token(message, force)
 
 
 @memoized
 def _get_github():
-    login_or_token, password = get_github_credentials()
-    return Github(login_or_token=login_or_token, password=password)
+    token = get_github_token()
+    return Github(login_or_token=token, password=None)
 
 
 @memoized
 def _github_auth_provided():
-    return bool(get_github_credentials()[0])
+    return bool(get_github_token()[0])
```

Comment:
I think this has a bug
```suggestion
    return bool(get_github_token())
```

Reply:
+1

Reply:
Thanks for catching this.